src/app/_models/depublication.ts

```diff
@@ -0,0 +1,33 @@
+export enum SortDirection {
+  UNSET = 'UNSET',
+  ASC = 'ASC',
+  DESC = 'DESC'
+}
+
+export class SortParameter {
+  field: string;
+  direction: SortDirection;
+}
+
+export enum DepublicationStatus {
+  PENDING = 'PENDING',
+  DEPUBLISHED = 'DEPUBLISHED'
+}
+
+export interface RecordPublicationInfo {
+  id: string;
+  recordId: string;
+  depublicationStatus: DepublicationStatus;
+  depublicationDate?: string;
+}
+
+export interface SortHeaderConf {
+  fieldName: string;
+  translateKey: string;
+  cssClass?: string;
+}
+
+export interface SortHeaderGroupConf {
+  cssClass: string;
+  items: Array<SortHeaderConf>;
+}
```

Comment:
Same as above. I assume that the idea is to make this generic eventually?

Reply:
Which the optional cssClass?  Exactly.